src/app/app.component.ts

```diff
@@ -1,15 +1,23 @@
-import { ChangeDetectionStrategy, Component } from '@angular/core';
-import { ProductComponent } from './product/product.component';
-import { CardListComponent } from './card/cardList.component';
+import {ChangeDetectionStrategy, Component} from '@angular/core';
+import {ProductComponent} from './product/product.component';
+import {CardListComponent} from './card/cardList.component';
+import {InputOutputComponent} from "./Exemple/input-output/input-output.component";
+import {AppMenuComponent} from "../AppMenu/app-menu/app-menu.component";
+import {RouterOutlet} from "@angular/router";
 
 @Component({
-  selector: 'app-root',
-  templateUrl: './app.component.html',
-  styleUrl: './app.component.css',
-  changeDetection: ChangeDetectionStrategy.OnPush,
-  standalone: true,
-  imports: [ProductComponent, CardListComponent],
+    selector: 'app-root',
+    templateUrl: './app.component.html',
+    styleUrl: './app.component.css',
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    standalone: true,
+    imports: [ProductComponent, CardListComponent, InputOutputComponent, AppMenuComponent, RouterOutlet],
 })
 
 export class AppComponent {
+    protected texts: string[] = ['coucou', 'bonjour', 'salut'];
```

Comment:
```suggestion
    protected texts = ['coucou', 'bonjour', 'salut'];
```
avec l'inférence de type il n'y a pas besoin de le préciser